config/config/wc-config.yaml

```
@@ -290,6 +290,8 @@ gatekeeper:
   allowUserCRDs:
     enabled: false
     enforcement: deny
+    # The name of the user specified in /etc/kubernetes/admin.conf kubeconfig file of the control plane nodes
```

Maybe we can add a comment that this is primarily for kubespray(?)
```suggestion
    # The name of the user specified in /etc/kubernetes/admin.conf kubeconfig file of the control plane nodes
    # Necessary if Kubespray is used for managing the k8s cluster.
```